strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-npm-layout-provider/src/test/java/org/carlspring/strongbox/artifact/generator/NpmPackageGenerator.java

```diff
@@ -66,6 +67,15 @@ public NpmPackageGenerator in(Path path)
         return this;
     }
 
+    @Override
+    public Path generateArtifact(URI uri,
+                                 int size)
+            throws IOException
+    {
+        this.in(Paths.get(uri));
```

Comment:
@yugander-krishan-singh you should provide `public NpmPackageGenerator(Path basedir)` constructor instead of doing `this.in(Paths.get(uri))` here.